tests/e2e/workloads/test_data_consistency.py

```diff
@@ -38,7 +39,11 @@ def test_log_reader_writer_parallel(project, tmp_path):
     # we need to mount the volume on every worker node, so RWX/cephfs
     pvc_dict["metadata"]["name"] = "logwriter-cephfs-many"
     pvc_dict["spec"]["accessModes"] = [constants.ACCESS_MODE_RWX]
-    pvc_dict["spec"]["storageClassName"] = constants.CEPHFILESYSTEM_SC
+    if storagecluster_independent_check():
+        sc_name = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_CEPHFS
+    else:
+        sc_name = constants.CEPHFILESYSTEM_SC
+    pvc_dict["spec"]["storageClassName"] = sc_name
```

Comment:
You can do it with one line
```pvc_dict["spec"]["storageClassName"] = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_CEPHFS if storagecluster_independent_check() else sc_name = constants.DEFAULT_EXTERNAL_MODE_STORAGECLASS_CEPHFS```

Reply:
Yea, for a better readability I have not added this way.